src/components/override-sdk/infra/View/View.tsx

```diff
@@ -26,10 +26,19 @@ const NO_HEADER_TEMPLATES = [
   'Confirmation'
 ];
 
+declare const PCore: any;
 export default function View(props) {
   const { children, template, getPConnect, mode, visibility, name: pageName } = props;
   let { label, showLabel = false } = props;
 
+  // call the locale reference api when we toggle the languge from English to Welsh
+  PCore.getPubSubUtils().subscribe('languageToggleTriggered', locale => {
+    if(!locale.localeRef.includes(props.localeReference)) {
```

Comment:
While publishing event you have passed localeRef=[], so i don't see any point of using if block.